### PR TITLE
Get started banner issue fixed

### PR DESCRIPTION
**Pull request by kodcuserkan**
Fixed for logged-in users

### CODE REVIEW

File: src/common/components/intro/index.tsx [hunk 1]
```diff
@@ -19,7 +19,17 @@ interface Props {
 export default (props: Props) => {
     const hideIntro = () => props.hideIntro();
 
-    if (!props.global.intro) {
+    const [isLoggedIn, setIsLoggedIn] = useState(false)
+
+    useEffect(() => {
+        const verification = localStorage.getItem("ecency_active_user")
```
Comment (feruzm) [inline src/common/components/intro/index.tsx:25]: I don't think you would need localStorage, you can use activeUser with props, no? Check `entry-index.tsx`

**Review by feruzm — CHANGES_REQUESTED** [review]

File: src/common/components/intro/index.tsx (hunk 1 above)
Comment (kodcuserkan) [reply src/common/components/intro/index.tsx:25]: Ok will fix

**Comment by kodcuserkan** [comment]
I did the changes used activeUser props to get current active user.

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**